packages/components/tests/dummy/app/templates/components/alert.hbs

```diff
@@ -0,0 +1,844 @@
+    >
+      <:actions as |A|>
+        <A.Button @text="Action" @color="secondary" />
+      </:actions>
+    </Hds::Alert>
+    <Hds::Alert
+      @type="inline"
+      @color="warning"
+      @title="With multiple actions passed as yielded components"
+      @description="Lorem ipsum dolor sit amet, consectetur adipiscing elit."
+    >
+      <:actions as |A|>
+        <A.Button @text="Secondary" @color="secondary" />
+        <A.Button @icon="plus" @text="Tertiary" @color="tertiary" />
+        <A.Link::Standalone @icon="plus" @text="Standalone" href="#" @color="secondary" />
+      </:actions>
+    </Hds::Alert>
+    <Hds::Alert
+      @type="inline"
+      @color="warning"
+      @title="With content yielded in the 'actions' named block"
+      @description="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua."
+    >
+      <:actions>
+        <DummyPlaceholder @text="some generic content" @height="50" @background="#eee" />
+      </:actions>
+    </Hds::Alert>
+    <Hds::Alert
+      @type="inline"
+      @color="warning"
+      @title="With actions and custom content in the 'actions' named block"
+      @description="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua."
+    >
+      <:actions>
+        <div>
+          <div class="dummy-alert-sample-custom-actions__actions">
+            <Hds::Button @text="Action" @color="secondary" />
+            <Hds::Link::Standalone @icon="plus" @text="Action" href="#" @color="secondary" />
+          </div>
+          <span class="dummy-alert-sample-custom-actions__text">This for example could be extra text, specific for a
+            special use case.</span>
+        </div>
+      </:actions>
+    </Hds::Alert>
+  </div>
+
+  <h4 class="dummy-h4">Dismiss</h4>
+  <div class="dummy-alert-sample-grid dummy-alert-sample-grid--wide-content">
+    <Hds::Alert
+      @type="inline"
+      @color="critical"
+      @title="Without the dismiss button (default)"
+      @description="Lorem ipsum dolor sit amet, consectetur adipiscing elit."
+    />
+    <Hds::Alert
+      @type="inline"
+      @color="critical"
+      @title="With the dismiss button"
+      @description="Lorem ipsum dolor sit amet, consectetur adipiscing elit."
+      {{! TODO: understand if we can use a generic helper - see https://hashicorp.slack.com/archives/C11JCBJTW/p1648751235987409 }}
+      @onClose={{this.noop}}
+    />
+    <Hds::Alert
+      @type="inline"
+      @color="critical"
+      @icon=""
+      @title="With the dismiss button and no icon"
+      @description="Lorem ipsum dolor sit amet, consectetur adipiscing elit."
+      @onClose={{this.noop}}
+    />
+    <Hds::Alert
+      @type="inline"
+      @color="critical"
+      @description="With the dismiss button and no title"
+      @onClose={{this.noop}}
+    />
+  </div>
+
+</section>
+
+<section>
```

Comment:
Please update this section to be more aligned with this: https://design-system-components-hashicorp.vercel.app/components/dropdown#accessibility

Reply:
done.